sdk/nodejs/src/client.ts

```diff
@@ -146,6 +146,7 @@ export class DevCycleClient {
                 if (err instanceof UserError) {
                     throw err
                 }
+                this._isInitialized = true
```

Comment:
why should this be `true`? also should it be set before the throw?

Reply:
is `_isInitialized` being used anywhere else in the SDK now?

Reply:
It's `true` because `isInitialized` represents either "successfully received config" as well as "successfully started client but didnt receive config".  The throw only happens if the user did something wrong when calling initialize, so if that happens then nothing will start up anyway and theres no reason to set `isInitialized`

Reply:
and `_isInitialized` is only being used in the `track` method now

Reply:
should that be using `hasConfig` as well?

Reply:
kinda questioning if there is a purpose to `isInitialized` anymore?